main/bamboo/bamboo.go

```diff
@@ -184,11 +184,11 @@ func listenToMarathonEventStream(conf *configuration.Configuration, sub api.Even
 	client := &http.Client{}
 	client.Timeout = 0 * time.Second
 
-	for _, marathon := range conf.Marathon.Endpoints() {
-		ticker := time.NewTicker(1 * time.Second)
-		eventsURL := marathon + "/v2/events"
-		go func() {
-			for _ = range ticker.C {
+	ticker := time.NewTicker(1 * time.Second)
+	go func() {
```

Comment:
LGTM. Though I'd be fine with making a larger change to handle errors explicitly (if conf.Marathon.Endpoints() does return nil here, I think it panics?)